app/src/main/res/values/strings.xml

```diff
@@ -151,6 +151,55 @@
     <!-- Item in Preferences that shows the about page. Parameter 1 is the app name, i.e. Firefox Focus/Firefox Klar. -->
     <string name="preference_about">About %1$s</string>
 
+    <!-- Header for the list of installed search engines -->
+    <string name="preference_search_installed_search_engines">Installed search engines</string>
+
+    <!-- Action for restoring the default list of search engines -->
+    <string name="preference_search_restore">Restore default search engines</string>
+
+    <!-- Action for adding an additional search engine to the list -->
+    <string name="preference_search_add">Add another search engine</string>
+    <string name="preference_search_remove">Remove</string>
+
+    <!-- Clickable text in tutorial to open a dialog to manually add a search engine -->
+    <string name="tutorial_search_manually_add">MANUALLY ADD</string>
+    <string name="tutorial_search_cancel">CANCEL</string>
+
+    <string name="tutorial_search_title">Add search engine</string>
+    <string name="tutorial_search_step1">1. Go to the search engine homepage.</string>
+    <string name="tutorial_search_step2">2. Tap into the search engine\'s search field.</string>
+    <!-- An alternative string for Step 2 of the tutorial -->
+    <string name="tutorial_search_step2_alternative">2. Long press the search engine\'s search field.</string>
+    <string name="tutorial_search_step3">3. Tap the magnifying glass icon above the keyboard to add the search engine.</string>
+    <!-- Clickable action text that shows when user long-presses on a search engine search field (this is similar to paste or copy when long-pressing a text field) -->
+    <string name="action_option_add_search_engine">ADD SEARCH ENGINE</string>
+
+    <!-- An alternative string for Step 3 of the tutorial. Parameter 1 is &action_option_add_search_engine, which is the action text that the tutorial is referring to -->
```

Comment:
nit: &action_option_add_search_engine looks like you are referencing a Gecko string because of the & :)